src/components/FileManager/ToggleHeader.tsx

```diff
@@ -0,0 +1,98 @@
+/*!
```

Comment:
this component looks like a generic UI components. Should it be in style-guide perhaps?